lib/ecto/query.ex

```diff
@@ -353,7 +353,7 @@ defmodule Ecto.Query do
   """
 
   defstruct [prefix: nil, sources: nil, from: nil, joins: [], aliases: %{}, wheres: [], select: nil,
-             order_bys: [], limit: nil, offset: nil, group_bys: [], updates: [],
+             order_bys: [], limit: nil, offset: nil, group_bys: [], unions: [], updates: [],
```

Comment:
Proposal: let's call this field "combines" or "combinations"? This way we can also add [intersect/except](https://www.postgresql.org/docs/8.3/static/queries-union.html) in the feature without adding more new fields.

Reply:
Good idea.